src/library_glfw.js

```diff
@@ -1228,7 +1228,8 @@ var LibraryGLFW = {
   glfwGetMonitors: function(count) {
     setValue(count, 1, 'i32');
     if (!GLFW.monitors) {
-      GLFW.monitors = allocate([1, 0, 0, 0], 'i32', ALLOC_NORMAL);
+      GLFW.monitors = {{{ makeMalloc('glfwGetMonitors', Runtime.POINTER_SIZE) }}};
+      setValue(GLFW.monitors, 1, 'i32');
```

Comment:
(this looks like from the other PR, which should land first I guess?)

Reply:
Sorry i tried to stack this on top of the other PR but as usual I didn't manage it.  I blame github for not making it easy to do this.